tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
Shorter, simpler, more readable, easier to understand, more straightforward, not bound to implementation details that don't matter for that test, not checking things that are not relevant for that test, easier to maintain, … `assertEquals` on the other side gave me as much trouble as using `==` gave us. We agree on the later, don't we? `assertEquals` does not compare arrays the way you expect. It does not compare strings the way you expect. It compares way to many non-relevant implementation details when used on objects. Slow. Unclear. Clunky. Unpredictable in wort cases.